pylint/message/message_handler_mix_in.py

```diff
@@ -347,18 +348,29 @@ def add_one_message(
             )
         )
 
-    def print_full_documentation(self, stream=None):
-        """output a full documentation in ReST format"""
-        if not stream:
-            stream = sys.stdout
+    def _get_checkers_infos(self) -> dict:
+        by_checker: dict = {}
+        for checker in self.get_checkers():
+            if checker.name != "master":
```

Comment:
Let's use a constant for `master`.

Reply:
I will do a different PR for that. Adding something in `constant.py` then changing "master" to something else and see what breaks.

Reply:
There wasn't that much changes after all, I'll include it in this PR.